app/Http/Controllers/RegisterController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+ 
+namespace App\Http\Controllers;
+ 
+use Illuminate\Http\Request;
+use App\Models\User;
+use Illuminate\Support\Facades\Auth;
+use Hash;
+class RegisterController extends Controller
+{
+    public function SignUp(Request $request)
```

Comment:
```suggestion
    public function signUp(Request $request)
```
name of the function should be written in `camelCase`

Reply:
this function would be better named as a `register`